statsmodels/base/model.py

```diff
@@ -510,6 +510,149 @@ def hess(params, *args):
         mlefit.mle_settings = optim_settings
         return mlefit
 
+    def _fit_zeros(self, keep_index=None, start_params=None, return_auxiliary=False,
```

Comment:
If this is a high enough priority to put in the base class, then we should also be discussing `_get_start_params`.

Reply:
This adds new functionality
`_get_start_params` is a backwards incompatible refactoring that needs 3 authors to agree.

(I'm not sure yet `_fit_zeros` and `_fit_collinear` should be inherited by tsa models, see PR description)

Reply:
> _get_start_params is a backwards incompatible refactoring

False.  It is entirely backwards-compatible, as it just takes code currently present in `LikelihoodModel.fit` and makes it into a helper method that subclasses can override (instead of needing to override all of `fit`).  This can be discussed elsewhere.

> that needs 3 authors to agree.

This policy is new to me.

> (I'm not sure yet _fit_zeros and _fit_collinear should be inherited by tsa models, see PR description)

I'll give it a read.  As you can tell, the first pass was mostly for readability.

Reply:
> 3 authors

The authors don't have to agree necessarily, but they will inherit any base methods, which requires adjusting the existing pattern.

https://github.com/statsmodels/statsmodels/pull/4314#issuecomment-373847207
Skipper, Chad, Kerby and I used different methods or attributes, with most likely inconsistent signatures. (that's for the parts of statsmodels that I checked.
(It's still an inconsistency that should be removed, but it doesn't go without adjustment cost.)